src/js/video.js

```diff
@@ -129,7 +129,7 @@ setup.autoSetupTimeout(1, videojs);
  *
  * @type {String}
  */
-videojs.VERSION = '__VERSION__';
+videojs.VERSION = require('../../package.json').version;
```

Comment:
It would mean that anyone who wants to require videojs will need to use rollup or whatever to exclude everything.
this is also the reason for removing browserify-versionify. It makes it bundler agnostic.